cmd/livenessprobe/main.go

```diff
@@ -131,6 +134,11 @@ func main() {
 		klog.Error("only one of `--metrics-address` and `--http-endpoint` can be explicitly set.")
 		os.Exit(1)
 	}
+	if *logFormatJSON {
+		jsonFactory := json.Factory{}
+		logger, _ := jsonFactory.Create(logsapi.LoggingConfiguration{Format: "json"})
+		klog.InfoS("logging livenessprobe",logger)
```

Comment:
I think this is just logging `logger` but not telling klog to use it for logging

Reply:
@mauriciopoppe ok i will re check this, if it's possible could you provide me pointer on this.

Reply:
I wrote some pointers in https://github.com/kubernetes-csi/node-driver-registrar/issues/228#issuecomment-1281453724

Reply:
sure i will check that thanks